proposal.md

```diff
@@ -0,0 +1,48 @@
+# Preamble
+
+Title: nft-barter Proposal document 
+
+## Abstract 
+This nft-barter is an open market where you can mint, buy, sell and swap NFTs within collection. The mint, buy and sell funtionality will be
```

Comment:
what's the importance of 'within collection'? Does this mean e.g. cryptokittie for another cryptokittie only?

Reply:
for now, we were just aiming for that. 
We are working on finding a way to swap outside intra-collections as well.  We might have to think of an off-chain solution. But it's still a work in progress.

Reply:
okay makes sense. Maybe we can add a scope-of-work section?

Reply:
that's a very good idea.